sdk/go/common/resource/urn.go

```diff
@@ -110,19 +110,9 @@ func (urn URN) IsValid() bool {
 	urn = urn[len(URNPrefix):]
```

Comment:
Is this line necessary?

Reply:
No we could cut this down to just the SplintN check now because we're not actually looking at any of the fields.